compiler/circle-eval-diff/README.md

```diff
@@ -0,0 +1,46 @@
+# circle-eval-diff
+
+_circle-eval-diff_ compares inference results of two circle models.
+
+## Use cases
+
+1. _circle-eval-diff_ can be used to evaluate reconstruction errors of quantized models.
+2. _circle-eval-diff_ can be used to verify optimization (or any kind of value-preserving conversion) is safe.
+
+## Usage
+
+Run circle-eval-diff with the following arguments.
+
+--first_input_model: first model to compare (.circle).
+--second_input_model: second model to compare (.circle).
+--first_input_data: input data for the first model (.h5, directory). Random data will be used if this argument is not given.
+--second_input_data: input data for the second model (.h5, directory). Random data will be used if this argument is not given.
+--input_data_format: input data format (h5 (default), directory).
+--metric: metric to compare inference results (MAE (default), etc).
+
+```
+$ ./circle-eval-diff
+  --first_input_model <first_input_model>
+  --second_input_model <second_input_model>
+  --first_input_data <first_input_data>
+  --second_input_data <second_input_data>
```

Comment:
I thought first model and second one have same inputs. But, it seems not, doesn't it?

Reply:
No, it doesn't. Models' input signature can be different (fp32, u8), so input_data can be different too.

Of course, you can give the same input_data if two models have the same input signature.